style.css

```diff
@@ -0,0 +1,294 @@
+/* stylelint-disable no-descending-specificity */
```

Comment:
- [ ] As said by the previous reviewer, Kindly make sure you do not disable the linters rules for they help your code to respond to the normal standards and make it readable.
`Hint: To solve the descending-specificity, you may consider write your CSS from top to bottom and your HTML is structured, a child would not styled before its parent` :smile: